aten/src/ATen/native/native_functions.yaml

```diff
@@ -3978,7 +3978,7 @@
 # missing the `pad_mode` and `center` arguments, which are taken care of at
 # `torch.functional.py`. They shall be moved here once we have mapping between
 # Python strings and C++ Enum in codegen.
-- func: stft(Tensor self, int n_fft, int? hop_length=None, int? win_length=None, Tensor? window=None, bool normalized=False, bool? onesided=None, bool? return_complex=None) -> Tensor
+- func: stft(Tensor self, int n_fft, int? hop_length=None, int? win_length=None, Tensor? window=None, bool normalized=False, bool? onesided=None, bool return_complex=True) -> Tensor
```

Comment:
the declarations also need to be modified at a few other sites for e.g., https://github.com/pytorch/pytorch/blob/a46d4212bf47e46a00ebf33f6d8df997c8b6aff0/torch/functional.py#L377 (likely cause of the current test failures)

Reply:
another one: https://github.com/pytorch/pytorch/blob/cf1f59452b05322425f606afcf6114b4b448b356/torch/_tensor.py#L448

Reply:
you could find other instances by grepping: `grep -rs "return torch.stft(input, n_fft, return_complex=True)"`

Reply:
you could find other instances by: grep -rs "bool? return_complex"

Reply:
done, I had to check for `istft` which also has `return_complex`